docker-compose.dev.yml

```diff
@@ -12,15 +14,16 @@ services:
       context: ./ui
       dockerfile: Dockerfile
     environment:
-      TASKCLUSTER_ROOT_URL: http://ingress
+      TASKCLUSTER_ROOT_URL: http://taskcluster
     command: start:docker
     volumes:
       - ./clients:/app/clients
       - ./services:/app/services
       - ./libraries:/app/libraries
-      - ./ui:/app/ui
       - ./generated:/app/generated
       - .all-contributorsrc:/app/.all-contributorsrc
+      - ./ui:/app/ui
+      - ui_node_modules:/app/ui/node_modules/
```

Comment:
this is to prevent local `ui/node_modules` that might have compiled binaries for host architecture interference with container modules.